src/algokit/cli/explore.py

```diff
@@ -28,17 +28,41 @@ class NetworkConfiguration(NetworkConfigurationRequired, total=False):
 
 
 GITPOD_URL = os.environ.get("GITPOD_WORKSPACE_URL")
+CODESPAE_NAME = os.environ.get("CODESPACE_NAME")
```

Comment:
Misspelt

Reply:
Fixed in b5ce9a2539d5edba2c0fd1ea8188372ce807f3c1